CMSgov/mint-app
# Easi 2619/discussion and reply user table migration

# EASI-2619

## Changes and Description

- Discussion and Discussion reply updated to use the user table
- Front end updated to use the user account instead of user Info
- Postman collection updated


## How to test this change
1. Bring the app up
2. Make sure you can create discussions and replies
3. Make sure you see the correct information about who asked and answered the questions.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
